cloud/pkg/cloudhub/cloudhub.go

```diff
@@ -29,26 +33,30 @@ func (a *cloudHub) Group() string {
 	return "cloudhub"
 }
 
+// Enable indicates whether enable this module
+func (a *cloudHub) Enable() bool {
+	return a.enable
+}
 func (a *cloudHub) Start() {
 	messageq := channelq.NewChannelMessageQueue()
 
 	// start dispatch message from the cloud to edge node
 	go messageq.DispatchMessage()
 
 	// start the cloudhub server
-	if hubconfig.Get().ProtocolWebsocket {
+	if hubconfig.Get().WebSocket.Enable {
 		// TODO delete second param  @kadisi
```

Comment:
Suggest delete the `Get` in this PR, and remove these todos.

Reply:
done